plugins/azure/recoveryService/recoveryVaultByokEncrypted.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     recommended_action: 'Modify Recovery Service vault and enable BYOK encryption.',
     link: 'https://learn.microsoft.com/en-us/azure/backup/encryption-at-rest-with-cmk',
     apis: ['recoveryServiceVaults:getRecoveryServiceVault', 'recoveryServiceVaults:listBySubscriptionId'],
+    realtime_triggers: ['microsoftrecoverservices:valuts:write', 'microsoftrecoverservices:valuts:delete'],
```

Comment:
```suggestion
    realtime_triggers: ['microsoftrecoverservices:vaults:write', 'microsoftrecoverservices:valuts:delete'],
```
vaults is misspelled everywhere and please confirm whether its 'microsoftrecoverservices' or 'microsoftrecoveryservices'